django-cloudlaunch/cloudlaunch/signals.py

```diff
@@ -1,9 +1,12 @@
 """App-wide Django signals."""
 from celery.utils.log import get_task_logger
 
+from django.contrib.auth.signals import user_logged_in
```

Comment:
I think the signal here should be the post_save signal from the User model, as shown here: https://simpleisbetterthancomplex.com/tutorial/2016/07/28/how-to-create-django-signals.html
so that a profile is created whenever a user is created, regardless of logged in status.

Reply:
https://github.com/CloudVE/djcloudbridge/pull/2/commits/637c16b2f1f1577e75dbb9f7bc24c7c47db9cee5 includes userprofile creation when a user is created, as well as when a user logs in. Keeping the creation when a user logs in avoids the necessity to go back and retroactively create profiles for all users.